closed/OpenJ9.gmk

```diff
@@ -228,11 +228,16 @@ ifeq (true,$(OPENJ9_ENABLE_DDR))
 $(call openj9_add_jdk_bin, java.base, j9ddr.dat)
 
 .PHONY : run-ddrgen
-$(OUTPUTDIR)/vm/j9ddr.dat : run-ddrgen
+$(OPENJ9_VM_BUILD_DIR)/j9ddr.dat : run-ddrgen
+
 run-ddrgen :
+ifeq (true,$(OPENJ9_ENABLE_CMAKE))
+	$(MAKE) -C $(OPENJ9_VM_BUILD_DIR) j9ddr
+else
 	export CC="$(CC)" CXX="$(CXX)" $(EXPORT_MSVS_ENV_VARS) \
 		&& $(MAKE) -C $(OUTPUTDIR)/vm/ddr -f run_omrddrgen.mk
 endif
+endif
```

Comment:
Comments on these `endif` lines would be helpful to match them with their opening conditions.